app/generated_resources.grd

```diff
@@ -343,7 +343,7 @@ are declared in tools/grit/grit_rule.gni.
       <if expr="enable_supervised_users">
         <part file="supervised_user_error_page_strings.grdp" />
       </if>
-
+      
```

Comment:
whitespace

Reply:
It looks like this whitespace eventually did get removed, because I can't see it in the current head of branch.

Reply:
Fixed in 319a1793b707cd9bf12907e1e8702d2d11f627fc.